sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -639,6 +659,65 @@ static inline void _nib_ft_remove(_nib_offl_entry_t *nib_offl)
 }
 #endif  /* GNRC_IPV6_NIB_CONF_ROUTER */
 
+#if GNRC_IPV6_NIB_CONF_MULTIHOP_P6C || defined(DOXYGEN)
+/**
+ * @brief   Creates or gets an existing authoritative border router.
+ *
+ * @pre `addr != NULL`
+ *
+ * @param[in] addr  Address of the authoritative border router.
+ *
+ * @return  An authoritative border router entry, on success.
+ * @return  NULL, if no space is left.
+ */
+_nib_abr_entry_t *_nib_abr_add(const ipv6_addr_t *addr);
+
+/**
+ * @brief   Removes an authoritative border router
+ *
+ * @pre `addr != NULL`
+ *
+ * @param[in] addr  Address of the authoritative border router.
+ */
+void _nib_abr_remove(const ipv6_addr_t *addr);
+
+/**
+ * @brief   Adds a prefix to the managed prefix of the authoritative border
+ *          router
+ *
+ * @pre `(abr != NULL) && (offl != NULL) && (offl->mode & _PL)`
+ *
+ * @param[in] abr   The border router.
+ * @param[in] offl  The prefix to add.
+ */
+void _nib_abr_add_pfx(_nib_abr_entry_t *abr, _nib_offl_entry_t *offl);
```

Comment:
const for `offl`, maybe for `abr` also?

Reply:
`abr` is manipulated by this function, `offl` can do.